pkgs/by-name/pl/platformsh/package.nix

```diff
@@ -50,6 +50,12 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  doInstallCheck = true;
+
+  postInstallCheck = ''
+    stat "''${!outputBin}/bin/${finalAttrs.meta.mainProgram}"
```

Comment:
The tester in passthru should be enough as it does kinda --version on that expected binary

Reply:
Ah ok but CI did not fail on the MR https://github.com/NixOS/nixpkgs/pull/309681 that changed `bin/platform` by `bin/platformsh`... maybe I don't know about passthru will document myself about it

Reply:
right it should fail with

```
install: cannot stat 'platformsh': No such file or directory
error: builder for '/nix/store/6rf9x6pxbmr9lwfxr71i9jdr65dzr262-platformsh-5.0.13.drv' failed with exit code 1
```